src/Conductors/ChecksRole.php

```diff
@@ -71,4 +91,28 @@ public function all($role)
 
         return $this->clipboard->checkRole($this->user, $roles, 'and');
     }
+
+    /**
+     * Check if the user has none of the given roles.
+     *
+     * @param  string  $role
+     * @return bool
+     */
+    public function none($role)
+    {
+        return $this->not($role);
+    }
+
+    /**
+     * Check if the user has none of the given roles.
+     *
+     * @param  string  $role
+     * @return bool
+     */
+    public function not($role)
```

Comment:
We don't need this method either.

I read this:

```php
Bouncer::is($user)->notA('subscriber', 'editor');
```

As "Is the user not a subscriber or an editor". No need for `not` and `none`.